src/couch/src/couch_httpd_auth.erl

```diff
@@ -186,6 +188,25 @@ proxy_auth_user(Req) ->
             end
     end.
 
+jwt_authentication_handler(Req) ->
+    case {config:get("jwt_auth", "secret"), header_value(Req, "Authorization")} of
+        {undefined, _} -> Req;
+        {_, undefined} -> Req;
+        {Secret, "Bearer " ++ Jwt} ->
```

Comment:
we can simplify the clauses here by adding a guard of `when Secret /= undefined`, so the previous two clauses can be removed.